system/scripts/core/calc_ability.cs

```diff
@@ -0,0 +1,65 @@
+﻿//--- Melia Script ----------------------------------------------------------
+// Ability Calculation Script
+//--- Description -----------------------------------------------------------
+// Functions that calculate ability-related values, such as properties.
+//---------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using Melia.Zone;
+using Melia.Zone.Scripting;
+using Melia.Zone.Skills;
+
+public class AbilityCalculationsScript : GeneralScript
```

Comment:
This has presumably become obsolete with my recent change to add the reinforce ability data to skills and the bonuses to the skill calculations.